packages/subgraph/tests/token/governance-erc20.test.ts

```diff
@@ -279,5 +278,61 @@ describe('Governance ERC20', () => {
       member.assertEntity();
       assert.entityCount('TokenVotingMember', 1);
     });
+
+    test('it should move delegation between members and end up deleting th member with no balances or delegated tokens', () => {
+      let memberOneAddress = ADDRESS_ONE;
+      let memberTwoAddress = ADDRESS_TWO;
+      let memberThreeAddress = ADDRESS_THREE;
+      let pluginAddress = ADDRESS_SIX;
```

Comment:
you don't need `pluginAddress` as the default is `CONTRACT_ADDRESS`, so you don't need to pass it.

Reply:
no, its not, I followed how its done on the other tests, and if you check the build function it does not include the plugin address